etna/datasets/tsdataset.py

```diff
@@ -278,27 +313,20 @@ def make_future(
         2021-07-04          33          38    NaN          73          78    NaN
         """
         self._check_endings(warning=True)
-        max_date_in_dataset = self.df.index.max()
-        future_dates = pd.date_range(
-            start=max_date_in_dataset, periods=future_steps + 1, freq=self.freq, closed="right"
-        )
-
-        new_index = self.raw_df.index.append(future_dates)
-        df = self.raw_df.reindex(new_index)
-        df.index.name = "timestamp"
+        df = self._expand_index(df=self.raw_df, freq=self.freq, future_steps=future_steps)
 
         if self.df_exog is not None and self.current_df_level == self.current_df_exog_level:
-            df = self._merge_exog(df)
+            # current implementation doesn't fail here if it doesn't fail in __init__, so check_regressors is redundant
+            df = self._merge_exog(df=df, check_regressors=False)
```

Comment:
I'm not sure we should touch this here, I could remove this. 

But we probably don't need this check here, it doen't fail here if it doesn't fail in `__init__`.

Reply:
It seems that you are right.